File: 1-UTN_PROGRA1_LABO1_SPD/1-Programacion1/Guia3_CICLOS_EXACTOS_YOUTUBE/EJ2_C++_CONTADOR/EJ2.cpp
#include <iostream>
using namespace std;

/* Hacer un programa que solicite 20 números y calcule y emita por pantalla
cuántos son positivos (mayores a cero). Se debe mostrar un solo valor: el
conteo final.*/

int main (){

  int n1, contador = 0;

  for(int x = 0; x < 20; x++)
  {
    cout << x + 1 << ") Ingrese un numero: ";
    cin >> n1;

    if(n1 > 0)
    {
      contador ++;
    }

  }

  cout << "Los numeros mayores a cero son: " << contador;




  return 0;
}

File: 1.1)PRO1_LAB1_SPD_MAT_ARSO/1-Programacion1/3.1)Guia3_CICLOS_EXACTOS_YOUTUBE/EJ2_CS/EJ2.cs
using System;

namespace practicas_clase1
{
    class Program
    {
        static void Main(String [] args)
        {   
            int n, contador = 0;

            for (int i = 0; i < 5; i++){
                Console.WriteLine("Ingrese el numero " + (i+1) + ": ");
                n = int.Parse(Console.ReadLine());

                if (n > 0)
                    contador ++;
            }
            Console.WriteLine("La cantidad de numeros positivos son: " + contador);
        }                     
    }
}